movable.lua

```diff
@@ -1006,6 +1006,7 @@ SlashCmdList[slashGlobal] = function(inp)
 	else
 		if(not _LOCK) then
 			for k, obj in next, oUF.objects do
+				if(obj.isNamePlate) then return end -- bail out on nameplates since we can't move those
 				if(not obj.disableMovement) then
```

Comment:
There might be other objects after the nameplate.

```suggestion
				if(not obj.disableMovement and not obj.isNamePlate) then
```